src/sass/components/_candidate.scss

```diff
@@ -1,8 +1,9 @@
-// scoped variables
-$smallFont: 1.0em;
-$mediumFont: 1.6em;
-$largeFont: 2em;
+
 .candidate {
+  // scoped variables
```

Comment:
I don't think we'll typically want to scope variables like this, but in any case, I don't believe they actually get scoped in Sass unless they're nested inside a selector.